src/layouts/AppHeader.tsx

```diff
@@ -83,91 +83,120 @@ export function AppHeader() {
   );
```

Comment:
The diff is ugly, but I wrapped the `HideOnScroll` component in a fragment, and then show the top banner if the mobile menu is not open